configure.ac

```diff
@@ -1140,6 +1140,16 @@ TS_CHECK_CRYPTO_SET_RBIO
 # Check for DH_get_2048_256
 TS_CHECK_CRYPTO_DH_GET_2048_256
 
+saved_LIBS="$LIBS"
+
+TS_ADDTO([LIBS], ["$OPENSSL_LIBS"])
+
+AC_CHECK_FUNCS([ \
+  X509_get0_signature \
+])
+
+LIBS="$saved_LIBS"
```

Comment:
Why do we have to save/restore `LIBS`? Are you adding `OPENSSL_LIBS` to `LIBS` just for the `AC_CHECK_FUNCS`?

Reply:
This pattern is used in 7cedf228 so I just used the same pattern.

Reply:
Actually, the original commit was 40310afc3ae52bb673d0178ce73f69ada4991736. So @jablko can you help to explain? Thanks.